spec/features/finding/index_filtering_spec.rb

```diff
@@ -30,9 +29,19 @@
   end
 
   def given_there_are_some_editions
-    create(:edition, :published, title: "Totally irrelevant")
-    @primary_organisation = { "content_id" => SecureRandom.uuid, "internal_name" => "Organisation 1" }
-    @organisation = { "content_id" => SecureRandom.uuid, "internal_name" => "Organisation 2" }
+    @primary_organisation = { "content_id" => User.first.organisation_content_id,
```

Comment:
Comparing this against the format specs (e.g. `news_story_spec`), perhaps it would be a good idea to introduce a 'tag' factory for these things?